billing/lib/billing-cron.js

```diff
@@ -0,0 +1,37 @@
+import { startOfLastMonth, startOfMonth, } from './util.js'
+
+/**
+ * @param {{
+ *   customerStore: import('./api.js').CustomerStore
+ *   customerBillingQueue: import('./api.js').CustomerBillingQueue
+ * }} ctx
+ * @returns {Promise<import('@ucanto/interface').Result>}
+ */
+export const handleCronTick = async ctx => {
+  const from = startOfLastMonth()
+  const to = startOfMonth()
```

Comment:
I think it's a good idea to pass seed time instead of making those utils effectful, also in theory you may run into two moth range if this happens to run at really unlucky time.

```suggestion
  const now = Date.now()
  const from = startOfLastMonth(now)
  const to = startOfMonth(now)
```